### PR TITLE
Publish WPT test report

### PR DESCRIPTION
**Pull request by sadym-chromium**
* Helper script `test-report/htmlWptReport.mjs` generates HTML report.
* GitHub actions publish ChromeDriver report to https://googlechromelabs.github.io/chromium-bidi.

### CODE REVIEW

**Review by thiagowfx — APPROVED** [review]

File: .github/workflows/wpt-mapper.yml [hunk 1]
```diff
@@ -45,29 +45,33 @@ jobs:
...
         uses: actions/upload-artifact@v3
         with:
           name: wpt-metadata-mapper
           path: wpt-metadata/mapper
+      - name: Upload WPT run report
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: wptreport-mapper
+          path: wptreport.json
```
Comment (thiagowfx) [inline .github/workflows/wpt-mapper.yml:74]: FYI it's also possible to add multiple paths at once, e.g.

  - name: Upload expectations and report
        if: ${{ always() }}
        uses: actions/upload-artifact@v3
        with:
          name: wpt-mapper
          path: wpt-metadata/mapper
          path: wptreport.json

https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions

...although, LGTM as is either way.

Comment (sadym-chromium) [reply .github/workflows/wpt-mapper.yml:74]: thanks for the update! Those artefacts are going to be used in different scenarios, so let's keep the split at least for now.

**Review by foolip — APPROVED** [review]

File: .github/workflows/wpt-chromedriver.yml [hunk 2]
```diff
@@ -50,34 +59,63 @@ jobs:
...
         if: ${{ always() }}
         uses: actions/upload-artifact@v3
         with:
-          name: logs
-          path: logs
+          name: wpt-chromedriver-artifacts
+          path: out
+      # The following steps should be done only for merges in `main`.
+      - name: Prepare Pages
+        #        if: github.ref == 'refs/heads/main'
```
Comment (thiagowfx) [inline .github/workflows/wpt-chromedriver.yml:102]: Remove if comment, also below (or add it)

File: .github/workflows/wpt-mapper.yml [hunk 3]
```diff
@@ -23,6 +23,9 @@ jobs:
           cache: npm
       - name: Install and build npm dependencies
         run: npm ci
+      - name: Setup dirs
+        run: |
+          mkdir -p out
```
Comment (thiagowfx) [inline .github/workflows/wpt-mapper.yml:28]:
```suggestion
      - name: Setup dirs
        run: mkdir -p out
```

File: test-report/htmlWptReport.mjs [hunk 4]
```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2021 Google LLC.
```
Comment (thiagowfx) [inline test-report/htmlWptReport.mjs:2]: 2023

**Review by thiagowfx — APPROVED** [review]

**Pull request merged**